tests/precook-node-modules.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var temp = require('temp').track();
+var runCommand       = require('./helpers/run-command');
+var path = require('path');
+
+var tmpDir = temp.mkdirSync();
+var root = process.cwd();
+var moveDirectory    = require('./helpers/move-directory');
```

Comment:
Seems like this file is most just 1 space (not space aligned), can this extra space be removed?